mmpose/models/heads/voxelpose_head.py

```diff
@@ -0,0 +1,158 @@
+# ------------------------------------------------------------------------------
+# Adapted from
+# https://github.com/microsoft/voxelpose-pytorch/blob/main/lib/models
+# Original licence: MIT License.
+# ------------------------------------------------------------------------------
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import HEADS
+
+
+@HEADS.register_module()
+class CuboidCenterHead(nn.Module):
+    """Get results from the 3D human center heatmap.
+
+    Args:
+        cfg (dict):
+            space_size (list[3]): The size of the 3D space.
+            cube_size (list[3]): The size of the heatmap volume.
+            space_center (list[3]): The coordinate of space center.
+            max_num (int): Maximum of human center detections.
+            max_pool_kernel (int): Kernel size of the max-pool kernel in nms.
+    """
+
+    def __init__(self, cfg):
+        super(CuboidCenterHead, self).__init__()
+        self.grid_size = torch.tensor(cfg['space_size'])
```

Comment:
Consider using `Module.register_buffer` for `grid_size`, `cube_size` and `grid_center`. In this case, lines like L46-48 can be ommited.

Reply:
OK